ur_bringup/launch/ur_moveit.launch.py

```diff
@@ -406,7 +406,7 @@ def generate_launch_description():
         executable="rviz2",
         name="rviz2",
         output="log",
-        arguments=["-d", rviz_config_file],
+        arguments=["-d", ""],
```

Comment:
Why shouldn't we provide default rviz config?

Reply:
I don't expect it to get merged like this but I find loading the RViz config causes a simulation failure somehow. Probably timing-related. We can revert this commit.